gtsfm/runner/run_scene_optimizer_1dsfm.py

```diff
@@ -0,0 +1,51 @@
+import argparse
```

Comment:
Would you mind adding a module-level docstring describing the file contents?

Reply:
Done